task/06-conditions-n-loops-tasks.js

```diff
@@ -270,7 +317,7 @@ function isCreditCardNumber(ccn) {
  *   165536 (1+6+5+5+3+6 = 26,  2+6 = 8) => 8
  */
 function getDigitalRoot(num) {
-    throw new Error('Not implemented');
+    return num < 10 ? num : getDigitalRoot(+num.toString().split('').reduce((a, b) => Number(a) + (Number(b))));
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :